src/collections/domain/dtos/CollectionDTO.ts

```diff
@@ -3,6 +3,17 @@ export interface CollectionDTO {
   name: string
   contacts: string[]
   type: CollectionType
+  affiliation?: string
+  description?: string
+  metadataBlockNames?: string[]
+  facetIds?: string[]
+  inputLevels?: CollectionInputLevelDTO[]
+}
```

Comment:
Since we are only supporting S3 storage for now for file uploading, we can exclude the storage field from the use case and DTO for now, and add it in the future. What do you think?

Reply:
I think that's the correct way of doing it 👍🏼